File: app/src/main/java/konradrutkowski/com/tapapp/activities/MainActivity.java
package konradrutkowski.com.tapapp.activities;

import android.app.FragmentTransaction;
import android.content.pm.ActivityInfo;
import android.os.Bundle;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;

import konradrutkowski.com.tapapp.R;
import konradrutkowski.com.tapapp.fragments.PlacesListFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);

        setContentView(R.layout.activity_main);
        ActionBar actionBar = getSupportActionBar();
        if (actionBar != null) {
            actionBar.setTitle("Places");
        }
        initListPlacesFragment();
    }


    private void initListPlacesFragment() {
        FragmentTransaction ft = getFragmentManager().beginTransaction();
        PlacesListFragment placesListFragment = new PlacesListFragment();
        ft.add(R.id.fragmentContainer, placesListFragment, "LIST").commit();
    }
}

File: app/src/main/java/konradrutkowski/com/tapapp/activities/MainActivity.kt
package konradrutkowski.com.tapapp.activities

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import konradrutkowski.com.tapapp.R
import konradrutkowski.com.tapapp.fragments.PlacesListFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.container_for_fragment)

        savedInstanceState ?: initListPlacesFragment()
    }

    private fun initListPlacesFragment() {
        val transaction = fragmentManager.beginTransaction()
        transaction.add(R.id.fragmentContainer, PlacesListFragment(), "LIST").commit()
    }
}


